tensorflow/python/layers/utils.py

```diff
@@ -195,19 +195,7 @@ def smart_cond(pred, fn1, fn2, name=None):
   Raises:
```

Comment:
can't you just do:

smart_cond = control_flow_ops.smart_cond

?

Reply:
I did consider that before, but I think current way could make the user/developer access the function doc more easily. I also referred to some other wrapper codes in the project such as  https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/tpu/python/tpu/tpu.py#L499.

Reply:
Shouldn't the doc be the same, regardless of whether you copy it over or just use a function alias?  The docstring copy is a DRY violation

Reply:
I got it. Thanks for pointing out. I have submitted a commit to change to function alias assignment.